README.md

```diff
@@ -43,7 +43,7 @@ Anubis is brought to you by sponsors and donors like:
 
 ## Overview
 
-Anubis [weighs the soul of your connection](https://en.wikipedia.org/wiki/Weighing_of_souls) using a proof-of-work challenge in order to protect upstream resources from scraper bots.
+Anubis is a Web AI Firewall Utility that [weighs the soul of your connection](https://en.wikipedia.org/wiki/Weighing_of_souls) using one or more challenges in order to protect upstream resources from scraper bots.
```

Comment:
I know this PR already merged, but I happened across it, and my first thought about this sentence is that it sounds like Anubis is a firewall that itself uses AI.

Reply:
🤫 it's part of how I trick VC bros to give me money for an anti-ai project